services/brig/src/Brig/Options.hs

```diff
@@ -439,9 +439,10 @@ data Settings = Settings
     --   field names and values), should be in sync
     --   with Spar
     setRichInfoLimit :: !Int,
-    -- | Default locale to use
-    --   (e.g. when selecting templates)
-    setDefaultLocale :: !Locale,
+    -- | Default locale to use when selecting templates
+    setDefaultTemplateLocale :: !Locale,
+    -- | Default locale to use for users
+    setDefaultUserLocale :: !Locale,
```

Comment:
I like the names you picked.

Haven't checked the context, so some or all of these may already be taken care of.   (Maybe the haddocks could be more explicit about things, and also explain what the difference is between the two?)

- In order to make upgrades for on-prem customers easier, you should only add optional config fields if possible, so make sure this parses if either or both are missing.
- Make the actual value (hard-wired default or not) of `setDefaultTemplateLocale` the default value of `setDefaultUserLocale`, and
- `en` the default for `setDefaultTemplateLocale`.
- Rationale for the last three points: you're using `setDefaultUserLocale` a lot in lieu of the deleted `setDefaultLocale`, the latter had a hard-wired default of `en`, and was primarily used for `setDefaultTemplateLocale`.